src/questions/chapter5.yaml

```diff
@@ -0,0 +1,1239 @@
+- uuid: bd6b88a6-ea79-4745-8a9e-3fcccf36a7ab
+  type: MultipleChoice
+  content: The feature in object-oriented programming that allows the same operation to be carried out differently, depending on the object, is 
+  hint: noHint
+  explanation:
+  answerKeys:
+    - B
+  topic: OOP
+  subTopics:
+    - Polymorphism
+  difficultyLevel: Low
+  choices:
+    - content: Inheritance
+      key: A
+      order: 0
+    - content: Polymorphism
+      key: B
+      order: 1
+    - content: Overriding
+      key: C
+      order: 2
+    - content: Overloading
+      key: D
+      order: 3
+
+- uuid: f02bba8b-1836-4052-abdb-cb58f6f30554
+  type: MultipleChoice
+  content: 
+    Consider the following two statements 
```

Comment:
`    Consider the following two statements `  --> `Consider the following two statements related to inheritance`